spec/request/api/v1/auth/sign_up_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+describe 'Auth', type: :request do
+  let(:user) { build(:user) }
+  let(:params) do
+    {
+      name: user.name,
+      email: user.email,
+      password: user.password,
+      password_confirmation: user.password
+    }
+  end
+
+  describe 'POST api/v1/auth' do
+    describe 'success' do
```

Comment:
Usually, this will be a context instead of a describe block. Something like:
`context 'when the request is succesful' do `